migrator/clone/clone_test.go

```diff
@@ -48,30 +48,26 @@ func setVersion(t *testing.T, ver *versionPair) {
 }
 
 func TestCloneMigration(t *testing.T) {
```

Comment:
With the flag being permanent `TestCloneMigration` and  `TestCloneMigrationRocksToPostgres` are effectively duplicates.  This test and `mock_central.go` need a fair amount of cleanup in this regard.  But all that should be done in a separate PR